### PR TITLE
[AUTO] performance data collection for inference schedule policy within AUTO plugin

### PR DESCRIPTION
**Pull request by yangwang201911**
### Details:
 - hardcoding optimal number of infer request to be half.
 - enable option to set the property `ov::intel_auto_schedule_policy`

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: samples/cpp/benchmark_app/main.cpp [hunk 1]
```diff
@@ -561,6 +561,11 @@ int main(int argc, char* argv[]) {
             device_config.insert(ov::hint::allow_auto_batching(false));
         }
 
+        if (!FLAGS_sp.empty()) {
+            slog::info << "Inference schedule policy is " << FLAGS_sp << slog::endl;
+            device_config["SCHEDULE_POLICY"] = FLAGS_sp;
```
Comment (ilya-lavrenov) [inline samples/cpp/benchmark_app/main.cpp:566]: why do we have API 1.0 style setting?

Comment (ilya-lavrenov) [reply samples/cpp/benchmark_app/main.cpp]: thanks for clarification

File: samples/cpp/benchmark_app/benchmark_app.hpp [hunk 2]
```diff
@@ -313,6 +317,9 @@ DEFINE_string(api, "async", api_message);
 /// @brief Number of infer requests in parallel
 DEFINE_uint64(nireq, 0, infer_requests_count_message);
 
+/// @brief inference schedule policy for MULTI
+DEFINE_string(sp, "", schedule_policy);
```
Comment (Wovchena) [inline samples/cpp/benchmark_app/benchmark_app.hpp:321]:
```suggestion
DEFINE_string(sp, "DEVICE_PRIORITY", schedule_policy);
```
If `DEVICE_PRIORITY` is the default, than use it

**Review by Wovchena — CHANGES_REQUESTED** [review]

File: samples/cpp/benchmark_app/benchmark_app.hpp (hunk 2 above)
Comment (yangwang201911) [reply samples/cpp/benchmark_app/benchmark_app.hpp:321]: sorry for confusion here. has been updated labels and drafted this PR.

**Pull request closed without merging**